components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml

```diff
@@ -326,6 +326,42 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /configs/homeRealmIdentifiers:
```

Comment:
/configs/home-realm-identifiers

Reply:
Addressed in https://github.com/wso2/identity-api-server/pull/260/commits/475f70efbd193afc7762fa8658b98f7cfedf290c